src/book/03-guides/04-general/04-documentation.mdx

```diff
@@ -0,0 +1,88 @@
+---
+section: Guides
+chapter: General
+title: Code Documentation
+description: How to properly document code.
```

Comment:
Major nitpick alert:
I wouldn't call this "properly documenting code" per se. It's more adding tags for documentation generation. We could make this a general guide for documentation, but that's not what this is. I wouldn't be against that tho

```suggestion
description: How to use documentation tags
```